LibreNMS/Modules/Wireless.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * Wireless.php
+ *
+ * -Description-
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @link       http://librenms.org
+ *
+ * @copyright  2021 Otto Reinikainen
+ * @author     Otto Reinikainen <otto@ottorei.fi>
+ */
+
+namespace LibreNMS\Modules;
+
+use App\Observers\ModuleModelObserver;
+use Illuminate\Support\Collection;
+use LibreNMS\DB\SyncsModels;
+use LibreNMS\Device\WirelessSensor;
+use LibreNMS\Interfaces\Module;
+use LibreNMS\Interfaces\Polling\WirelessAccessPointPolling;
+use LibreNMS\OS;
+use LibreNMS\RRD\RrdDefinition;
+
+class Wireless implements Module
```

Comment:
This module should probably be called WirelessController

Reply:
Renamed

Reply:
Renamed